python/lsst/ap/verify/ingestion.py

```diff
@@ -184,14 +184,22 @@ def _ingestRaws(self, dataset, workspace):
             The dataset on which the pipeline will be run.
         workspace : `lsst.ap.verify.workspace.Workspace`
             The location containing all ingestion repositories.
+
+        Raises
+        ------
+        RuntimeError
+            Raised if there are no files to ingest.
```

Comment:
Thanks for documenting this. We need more docstrings that call out known exceptions.